package.json

```diff
@@ -14,7 +14,7 @@
     "build": "babel src -d dist",
     "start": "node -- dist/index.js",
     "deploy-now": "now && now alias",
-    "deploy-dev": "now `sed 's/#.*$//;/^\\s*$/d;s/^/-e /' .env`"
+    "deploy-dev": "now `sed 's/#.*$//;/^\\s*$/d;s/^/-e /' .env` && now alias alzabot-test"
```

Comment:
`now dev` is for now 2.0 (serverless). It doesn't work for the old now 1.0 ('[serverful](https://zeit.co/guides/migrate-to-zeit-now)'). It would be possible to use a `now.dev.json` or something like that instead of using sed magic, but that doesn't work easily with running locally - it is easier to use .env when running locally. So I kept it like this.